### PR TITLE
Added the ability to copy ACL from one OU to another with one command

### PR DESCRIPTION
**Pull request by guzzd**

### CODE REVIEW

**Comment by PrzemyslawKlys** [comment]
Hi, thank you for this. How New-ADSite is different than New-ADReplicationSite that comes built-in to the ActiveDirectory module?

**Comment by guzzd** [comment]
> Hi, thank you for this. How New-ADSite is different than New-ADReplicationSite that comes built-in to the ActiveDirectory module?

Hi, when creating a new site one normally also want to add subnets and a site link, this does it in one go. It's easily done using the built-in AD module commands of course. I realize now that the command has some hard coded (sitelink name, site link cost and interval) things that one might want to set as parameters. Plus that it's a little bit overworked and don't qualify as a command.

**Comment by PrzemyslawKlys** [comment]
Any reason why you closed this?

File: Public/Copy-ADOUSecurity.ps1 [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<#
```
Comment (PrzemyslawKlys) [inline Public/Copy-ADOUSecurity.ps1:1]: I believe synopsis should be within function above cmdletbinging().

**Comment by PrzemyslawKlys** [comment]
Let me know when you will be ready with merging?

File: Public/Copy-ADOUSecurity.ps1 (hunk 1 above)
Comment (guzzd) [reply Public/Copy-ADOUSecurity.ps1]: Corrected this and added credential param.

File: Public/Copy-ADOUSecurity.ps1 [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            $oTargetOU.ObjectSecurity.AddAccessRule($oAce)
+        }
+
+        $oSourceOU.ObjectSecurity.Access | Where-Object { $_.IdentityReference -like "$sDomain\$sSourceAccount" }
+        $oTargetOU.ObjectSecurity.Access | Where-Object { $_.IdentityReference -like "$sDomain\$sDestAccount" }
+
+        if ($Execute) {
+            try {
+                $oTargetOU.CommitChanges()
+                Write-Output -InputObject "[Info] : Permissions commited"
```
Comment (PrzemyslawKlys) [inline Public/Copy-ADOUSecurity.ps1:76]: I would suggest changing Write-Output to Write-Verbose

File: Public/Copy-ADOUSecurity.ps1 [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+            } catch {
+                $ErrorMessage = $_.Exception.Message
+                Write-Warning -Message $ErrorMessage
+            }
+        } else {
+            Write-Output -InputObject "[Info] : Use the switch -Execute to apply..."
```
Comment (PrzemyslawKlys) [inline Public/Copy-ADOUSecurity.ps1:82]: Write-Verbose or Write-Warning if it's supposed to be visible always would be better choice.

**Pull request merged**